tests/sdk/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -xe
+echo "IN THE ENTRYPOINT"
```

Comment:
Aren't you missing something here ?

Reply:
The entrypoint and others will be filled with Another PR soon.
Please ignore that for now.

Reply:
How can I make sure that the code added in this PR works then ?

Reply:
What code you did you mean?

Reply:
There is executable file `sdk` and it is came from the package `sdk` the package include `main.go`.
when `./sdk` statement executed then, the relevant test cases are executed.

Reply:
Oh, nevermind, I'm blind...
Just remove the echo and we're good 👍

Reply:
Note that you can just set the binary as entrypoint if you wish, also. But maybe you want to extend this script ?

Reply:
yes. I have a plan to extend the script to support various kind of testing needs